src/mixins/appSettings.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+export default {
```

Comment:
Is this mixin needed? Can this code go in [src/store/app.module.js](https://github.com/cylc/cylc-ui/pull/1330/files#diff-d7b6802e371ab4383805859b39da1ae0320e4a80540d287cfbcc9db88c84e152)?

Reply:
I looked into making this a Vuex action but it needs access to `this.$vuetify` on the component instance

Reply:
Dammit! We should really collect these options, they are all managed differently ATM, but this suggests VueX isn't the place to collect them.